compiler/z/CMakeLists.txt

```diff
@@ -57,7 +57,6 @@ compiler_library(z
 	${CMAKE_CURRENT_LIST_DIR}/codegen/OMRCodeGenerator.cpp
 	${CMAKE_CURRENT_LIST_DIR}/codegen/snippet/PPA1Snippet.cpp
 	${CMAKE_CURRENT_LIST_DIR}/codegen/snippet/PPA2Snippet.cpp
-	${CMAKE_CURRENT_LIST_DIR}/codegen/snippet/XPLINKCallDescriptorSnippet.cpp
```

Comment:
Is there any particular reason behind not deleting the `XPLINKCallDescriptorSnippet.cpp` and `XPLINKCallDescriptorSnippet.hpp` files themselves? It looks like they're unused after this change.

Reply:
Right, I should have just said it explicitly in a PR comment. The commit body (aa4d2cf) explains that we cannot remove this until downstream projects stop compiling this file.

Reply:
Opened up eclipse/openj9#8739 to address this.